content/en/docs/guides/input_and_access.md

```diff
@@ -217,15 +250,18 @@ Takes an image that is located in an OCI registry and adds it as a resource.
       path: gcr.io/google_containers/echoserver:1.10
       repository: images/echo
 ```
-The target location of the image can be set with the `repository` field. Here the resulting image will be stored at `github.com/open-component-model/megacomponent/images/echo:1.10` and contained in the field `referenceName` in the component descriptor:
+
+The target location of the image after transporting to an OCI registry can be set with the `repository` field. Here the resulting image will be prefixed with the name of the component version, e.g. `github.com/open-component-model/megacomponent/images/echo:1.10`. This auto-prefix can be disabled by using a leading slash `/images/echo`. The resulting path is contained in the field `referenceName` in the component descriptor:
 
 ```yaml
-   - access:
-        ...
-        referenceName: github.com/open-component-model/megacomponent/images/echo:1.10
-      name: mega-image
-      version: 0.1.0
+spec:
+  resources:
+  - name: mega-image
+    version: 0.1.0
+    access:
       ...
+      referenceName: github.com/open-component-model/megacomponent/images/echo:1.10
+    ...
```

Comment:
Same as above

Reply:
@jensh007 I think you should then also remove this section?

Reply:
done- Thank You!